### PR TITLE
run_once_programs method

### LINKED ISSUE #22: Add support to run once off program
`/cr?t=[X,X,X,X,...]` endpoint where C is number of seconds

### PR DESCRIPTION
**Pull request by travisghansen**
Closes #22

### CODE REVIEW

**Comment by travisghansen** [comment]
@vinteo another release :)

**Comment by travisghansen** [comment]
Actually hold tight, gonna look into the `/jp` endpoint mentioned in the other thread real quick.

File: pyopensprinkler/__init__.py [hunk 1]
```diff
@@ -257,6 +257,16 @@ def set_water_level(self, level):
 
         return self._set_option("wl", level)
 
+    def run_once_program(self, station_times):
+        """Run once program"""
+        params = {"t": station_times}
```
Comment (vinteo) [inline pyopensprinkler/__init__.py:262]: Is it worth to to check the length of `station_times` with the number of stations?

Comment (travisghansen) [reply pyopensprinkler/__init__.py:262]: @vinteo it doesn't matter actually. For example I tested with my setup which has 24 stations with `controller.run_once_program([0, 60])`. If you wanted to skip stations you do need to explicitly set them, but the tail doesn't make any difference.

Comment (vinteo) [reply pyopensprinkler/__init__.py:262]: ok great, makes it easy

**Comment by travisghansen** [comment]
@vinteo go ahead and merge...not gonna address the start time stuff ATM.

**Pull request merged**